lib/rules/no-invalid-double-slash-comments/README.md

```diff
@@ -1,6 +1,6 @@
 # no-invalid-double-slash-comments
 
-Disallow double-slash comments (`//...`) which are not supported by CSS and [could lead to unexpected results](http://stackoverflow.com/a/20192639/130652).
+Disallow double-slash comments (`//...`) which are not supported by CSS and [could lead to unexpected results](https://stackoverflow.com/a/20192639/130652).
```

Comment:
Funny thing: it redirects to HTTP. Shall we use the final URL instead? https://stackoverflow.com/questions/12298890/is-it-bad-practice-to-comment-out-single-lines-of-css-with/20192639#20192639